x-pack/plugins/observability/public/components/shared/alert_search_bar/types.ts

```diff
@@ -13,6 +17,35 @@ export interface AlertStatusFilterProps {
   onChange: (id: string, value: string) => void;
 }
 
+export interface AlertSearchBarWithUrlSyncProps extends CommonAlertSearchBarProps {
+  urlStorageKey: string;
+}
+
+export interface KibanaDependencies {
+  data: {
+    query: {
+      timefilter: { timefilter: TimefilterContract };
+    };
+  };
+  notifications: { toasts: ToastsStart };
+  triggersActionsUi: {
+    getAlertsSearchBar: (props: AlertsSearchBarProps) => ReactElement<AlertsSearchBarProps>;
+  };
+}
+
+export type ObservabilityAlertSearchBarDependencies = KibanaDependencies;
+
+export interface Services {
+  timeFilterService: TimefilterContract;
+  AlertsSearchBar: (props: AlertsSearchBarProps) => ReactElement<AlertsSearchBarProps>;
+  errorToast: (error: Error, options: ErrorToastOptions) => Toast;
+}
+
+export interface ObservabilityAlertSearchBarProps
+  extends AlertSearchBarContainerState,
+    AlertSearchBarStateTransitions,
+    CommonAlertSearchBarProps {}
```

Comment:
Is the `{}` necessary?  Never seen that before in an `extends` clause.

Reply:
`ObservabilityAlertSearchBarProps` does not have any extra parameters than the 3 interfaces that it extends. So the `{}` is empty. Not sure if there is another way to write it.

Reply:
Ah.  In that case, try:

```
export type ObservabilityAlertSearchBarProps = 
  AlertSearchBarContainerState & 
  AlertSearchBarStateTransitions & 
  CommonAlertSearchBarProps;
```